ci/pipelines/deploy-to-test.yml

```diff
@@ -5301,6 +5371,38 @@ jobs:
           ENVIRONMENT: test-12
     <<: *put_success_slack_notification      
     <<: *put_failure_slack_notification
+
+  - name: smoke-test-webhooks-egress
```

Comment:
Need to add this as a passed param for push-webhooks-egress-to-staging-ecr so the push requires the smoke test to pass

Reply:
Yeah good eyes, I noticed this after applying it.